pegasus/src/forms.rb

```diff
@@ -2,6 +2,10 @@
 require 'cdo/regexp'
 require src_dir 'database'
 require lib_dir 'forms/pegasus_form_errors'
+require lib_dir 'forms/pegasus_form_validation'
+require 'active_support/core_ext/string/inflections'
 
-# Load forms
-Dir.glob(pegasus_dir('forms/*.rb')).each {|path| require path}
+# Autoload all classes in forms directory.
+Dir.glob(pegasus_dir('forms/*.rb')).each do |path|
+  autoload File.basename(path, '.rb').camelize, path
+end
```

Comment:
Nice!

Reply:
oh, I should mention that aside from being generally more efficient, this change was needed because declaring a `Sequel::Model` subclass immediately causes a database query to load its schema definition, which was causing our CI to fail on the initial `rake install` because the database hadn't yet been created at the time the `Form` class was initially `require`d during the Rails-application initialization process.

lazy-loading these models using `autoload` turned out to be the easiest incremental workaround.